docs/modules/opa/pages/usage.adoc

```diff
@@ -158,10 +158,10 @@ servers:
       config:
         resources:
           cpu:
-            min: '200m'
-            max: "2"
+            min: 250m
+            max: 500m
           memory:
-            limit: '2Gi'
+            limit: 512Mi
```

Comment:
I believe we also wanted to document the minimum of resources required to run a simple installation.

Here is an example: https://github.com/stackabletech/druid-operator/pull/444/files#diff-742ddb0f4a8446472c6e339a5f85170cf530742b3ca84072593cea5bddffb7c8

Reply:
Watch out! We need the <whatever> resources for every Node of the clusters (as it's a daemonset)
Special text needed

Reply:
https://github.com/stackabletech/opa-operator/pull/453/commits/116ba6016153d20a397d7ba0e17411046ea3be5a